bucket/sophiapp.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.0.82",
+    "description": "The most powerful open source tweaker for Windows 10 & 11.",
+    "homepage": "https://github.com/Sophia-Community/SophiApp/",
+    "license": "MIT",
+    "notes": [
+        "Be careful when using this app, you can cause some serious harm to your computer."
+    ],

```

Comment:
My opinion is that if this app has potential problem to **break the system**, then we shouldn't add this to the official buckets.

Reply:
I tested it again on my system, I was wrong about it harming my computer. It turns out, it just closes some processes such as, _explorer.exe_, and _everything.exe_ while it applies the tweaks that you want. 

I will remove will note now, consider this a false warning from my end. No need to worry.